backend/push/env.ts

```diff
@@ -1,14 +1,13 @@
 import * as dotenv from 'dotenv';
-import { getOsEnv } from './src/common/utils/path.helper';
 
 dotenv.config();
 
 export const env = {
     app: {
-        tokenSecret: getOsEnv('TOKEN_SECRET'),
-        port: getOsEnv('PUSH_APP'),
-        bootstrapAddress: getOsEnv('BOOTSTRAP_ADDRESS'),
-        consumerGroup: getOsEnv('CONSUMER_GROUP'),
-        notificationsTopic: getOsEnv('NOTIFICATIONS_TOPIC')
+        tokenSecret: 'mdzLa0RXxQX8mL9hFjctK9hCTDW4BQXHZXQZPPAKdD0LR6K0rB9rJrb6rM1otpWGXVABmxH0n4G1oFkp5Z03fGMXpvi3yeMq9jgxpnnNU6vP0j2UneEdm8Me1JK9qEQV',
+        port: '3003',
+        bootstrapAddress: '127.0.0.1:9092',
+        consumerGroup: 'notifiactions_group',
+        notificationsTopic: 'notifications'
```

Comment:
what was wrong with previous implementation?

Reply:
Oops, my bad, thanks for noticing